packages/dds/sequence/src/test/collections.intervalTree.spec.ts

```diff
@@ -59,14 +59,9 @@ class TestInterval implements ISerializableInterval {
 }
 
 describe("Collections.IntervalTree", () => {
-	let intervalTree: IntervalTree<TestInterval>;
-
-	beforeEach(() => {
-		intervalTree = new IntervalTree<TestInterval>();
-	});
-
 	describe(".map", () => {
 		it("Should run each node through the given function", () => {
+			const intervalTree = new IntervalTree<TestInterval>();
```

Comment:
This is an unrelated change, but I couldn't help myself from cleaning up this needless use of mutable state outside tests.